Source/Core/Core/Movie.cpp

```diff
@@ -1579,4 +1584,77 @@ void Shutdown()
   tmpInput = nullptr;
   tmpInputAllocated = 0;
 }
+
+void CheckWiiSaves()
+{
+  std::string save_path = Common::GetTitleDataPath(GetTitleId(), Common::FROM_SESSION_ROOT);
+  std::string user_save_path = Common::GetTitleDataPath(GetTitleId(), Common::FROM_CONFIGURED_ROOT);
+  if (IsRecordingInput())
+  {
+    // TODO: Check for the actual save data
+    SetClearSave(File::Exists(user_save_path + "banner.bin"));
```

Comment:
This logic seems inverted to me. The case where we are starting from a clear save is supposed to be the case where banner.bin doesn't exist.